packages/edit-site/src/components/global-styles/utils.js

```diff
@@ -222,11 +222,24 @@ function getValueFromCustomVariable( features, blockName, variable, path ) {
 	return getValueFromVariable( features, blockName, result );
 }
 
+/**
+ * Attempts to fetch the value of a theme.json CSS variable.
+ *
+ * @param {Object}   features  GlobalStylesContext config, e.g., user, base or merged. Represents the theme.json tree.
+ * @param {string}   blockName The name of a block as represented in the styles property. E.g., 'root' for root-level, and 'core/${blockName}' for blocks.
+ * @param {string|*} variable  An incoming style value. A CSS var value is expected, but it could be any value.
+ * @return {string|*|{ref}} The value of the CSS var, if found. If not found, the passed variable argument.
+ */
 export function getValueFromVariable( features, blockName, variable ) {
 	if ( ! variable || typeof variable !== 'string' ) {
 		if ( variable?.ref && typeof variable?.ref === 'string' ) {
 			const refPath = variable.ref.split( '.' );
 			variable = get( features, refPath );
+			// Presence of another ref indicates a reference to another dynamic value.
+			// Pointing to another dynamic value is not supported.
+			if ( ! variable || !! variable?.ref ) {
```

Comment:
do we need `.?` here since we have already checked that `variable` is defined?

Reply:
I left it in there since variable is reassigned above: `variable = get( features, refPath );`

Reply:
sorry, wasn't 100% clear. On line 240 we check again if it is defined with ` if ( ! variable ...`. I thought we would only need `.?` if we were only checking for `.ref`, eg. `if ( !! variable?.ref )` ?  ... it is Friday afternoon though, so I may be missing something 😄

Reply:
Ah sorry 🤦 was my Friday bad. 

Good question. I wanted to check for both just in case the ref pointed to a dead end, not just another ref.

I'll update the comment.